TestProjects/Playground/Playground.Forms.Uwp/Setup.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Activation;
+using MvvmCross.Core.Navigation;
+using MvvmCross.Core.ViewModels;
+using MvvmCross.Core.Views;
+using MvvmCross.Forms.Uwp;
+using MvvmCross.Forms.Uwp.Presenters;
+using MvvmCross.Platform;
+using MvvmCross.Platform.Logging;
+using MvvmCross.Platform.Platform;
+using MvvmCross.Uwp.Views;
+using Xamarin.Forms.Internals;
+using XamlControls = Windows.UI.Xaml.Controls;
+
+namespace Playground.Forms.Uwp
+{
+    public class Setup : MvxFormsWindowsSetup
+    {
+        private readonly LaunchActivatedEventArgs _launchActivatedEventArgs;
+
+        public Setup(XamlControls.Frame rootFrame, LaunchActivatedEventArgs e) : base(rootFrame, e)
+        {
+            _launchActivatedEventArgs = e;
+        }
+
+        protected override IMvxApplication CreateApp()
+        {
+            return new Core.App();
+        }
+
+        protected override IMvxLogProvider CreateLogProvider()
+        {
+            return new EmptyVoidLogProvider();
+        }
+
+        protected override IMvxWindowsViewPresenter CreateViewPresenter(IMvxWindowsFrame rootFrame)
+        {
+
+            Xamarin.Forms.Forms.Init(_launchActivatedEventArgs);
```

Comment:
This should be done in the base class to make it for users easy.

Reply:
Instead of creating application im using property FormsApplication of base class now. This will init xamarin forms